examples/ZMQPP/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ZMQPP)
+
+hunter_add_package(ZMQPP)
+
+find_package(ZMQPP CONFIG REQUIRED)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
> You wanna add c++11 flags in example CMakeLists.txt or I can raise error message but currently I have to check the CMAKE_CXX_FLAGS variable instead of CMAKE_CXX_STANDARD variable?

@tatraian This is up to you. Toolchains set `CMAKE_CXX_FLAGS` so you can check it. In general `CXX_STANDARD` can be checked too. Also this can be done for both in C++ code of the library instead of CMake code.

Set of `c++11` standard in local project should be done under condition `if(no_toolchains)` (both in example and top CMakeLists.txt of ZMQPP).

> Now as I see the CMakelists.txt of zmqpp the building of ZMQPP will failed if C++11 is not enabled (on linux).

This is okay since it is a requirement. Toolchain with name `default` must fail, I will exclude it from testing. Also adding `-std=c++11` may not help in all cases since old compilers from Travis environment may be missing some c++11 features anyway.